Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,23 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
+    system "./configure"
+    system "make"
+    bin.install "main/ch_track"
+    bin.install "main/ch_wave"
+    bin.install "main/pda"
+    bin.install "main/pitchmark"
+    bin.install "main/sig2fv"
+    bin.install "main/sigfilter"
+    bin.install "main/spectgen"
+    bin.install "main/wagon"
+  end
+
+  test do
+    assert_equal "Error reading ESPS file -", `ch_track 2>&1`.strip
```

Comment:
How about something like this:
```ruby
  test do
    # define some variables
    rate = 16000
    freq = 100
    dur = 5
    bn = 'sine'
    txtfile = "#{bn}.txt"
    wavfile = "#{bn}.wav"
    ptcfile = "#{bn}.ptc"

    # create 5 s, 16 kHz sampled, 100 Hz sine wave as ASCII file
    File.open(txtfile, 'w') do |f|
      scale = 2 ** 15 - 1
      f.puts Array.new(dur * rate) { |i| (scale * Math.sin(freq * 2 * Math::PI * i / rate)).to_i }
    end

    # convert to wav format using ch_wave
    %x(ch_wave #{txtfile} -itype raw -istype ascii -f #{rate} -o #{wavfile} -otype riff)

    # pitch tracking to est format using pda
    %x(pda #{wavfile} -shift #{1 / freq.to_f} -o #{ptcfile} -otype est)

    # extract one frame from the middle using ch_track, capturing stdout
    pitch = %x(ch_track #{ptcfile} -from #{freq * dur / 2} -to #{freq * dur / 2}).strip

    # should be 100 (Hz)
    assert_equal freq, pitch.to_i
  end
```
which creates a test file, and chains three EST binaries to verify the output.

I can commit this, then should I squash or would you?